lib/google/gax/grpc.rb

```diff
@@ -122,19 +122,22 @@ def create_stub(service_path,
                       interceptors: [])
         verify_params(channel, chan_creds, updater_proc)
         address = "#{service_path}:#{port}"
+        default_channel_args = { 'grpc.service_config_disable_resolution' => 1 }
```

Comment:
We have a few Google Cloud gems that create `GRPC::Core::Channel` objects because they need to set other channel args. Those gems will not benefit from this change. I would prefer to see `channel_args` added as an argument to `Google::Gax. create_stub` _and_ the GAPIC client constructor so we can pass the arguments instead of creating and passing a channel object.

Reply:
Once we have those two changes, we can update the Google Cloud gems to pass `channel_args` instead of creating a channel object.